### DISCUSSION TITLE
Is it possible to batch-convert DV videos into HDR10+?

### DISCUSSION THREAD

**Discussion by m1ndy** [R3S3t9999/DoVi_Scripts#20] [General]
I would like to do an HLG->PQ conversion along the way as that's required (?) for the HDR10 standard. I do have DaVinci Resolve Studio, if that helps with certain licensed elements in the pipeline.

I have a bunch of iPhone/Xiaomi 14 Ultra captured videos that I  would like  to convert to royalty free formats (for upload to Wikimedia, Flickr, etc). So the transfer involves:

Source (iPhone proprietary-encumbered) | Destination (royalty-free)
-- | --
4k 60fps (UHD, not DCI) | 4k 60fps
BT.2020 in HLG (WCG) | BT.2020 in HLG (WCG)
Frame-level dynamic HDR metadata in Dolby Vision profile 8.4 | Frame-level dynamic HDR metadata in HDR10+
HEVC | VP9 (open to AV1 as well but that's likely more nascent)
Bitrate is 31 Mb/s | Fine with higher bitrate to maintain same SSIM/PSNR
Audio is in AAC | Audio to Vorbis/OGG

I can do the rest of the codec conversions etc, but the HDR dynamic metadata is a conundrum.

Relevant links for more context from others:
1. https://video.stackexchange.com/questions/33886/how-to-convert-dolby-vision-hdr-to-hdr10?noredirect=1
2. https://www.reddit.com/r/hometheater/comments/eholm5/dolby_vision_to_hdr10_conversion/
3. https://github.com/quietvoid/dovi_tool/issues/23

**Comment by R3S3t9999** [comment]
Hi, the workflow to convert HLG to HDR10  hevc is 8-2-1

**Reply by m1ndy** [reply]
Will that be HDR10 or HDR10+?

What algo is used to convert from HLG to PQ there?

**Comment by R3S3t9999** [comment]
no AFAIK, you cannot convert DV metadata to HDR10+ so the output will be static HDR10... I'm not sure if [dovi_tool](https://github.com/quietvoid/dovi_tool) can convert profile 8.4 to profile 8.1, you'll have to read about it.

The script uses libplacebo for the HLG to HDR10 PQ conversion

**Comment by m1ndy** [comment]
Is there a way to convert DV metadata to HDR10+ using DaVinci Resolve?

**Comment by R3S3t9999** [comment]
No, but you can generate HDR10plus:
https://www.youtube.com/watch?v=FVSh3oGqfXY

**Comment by m1ndy** [comment]
That video is how I came across your github in the first place :) Thanks for making it! I learn a lot from that and your livestreams :)

Can DaVinci Resolve decode an existing DV dynamic data to then write it out as an HDR10+?

**Comment by R3S3t9999** [comment]
> existing DV dynamic data to then write it out as an HDR10+?

No.
